src/FakeItEasy.dotnet/project.json

```diff
@@ -18,8 +18,6 @@
   },
 
   "frameworks": {
-    "netstandard1.6": {
-      "imports": [ "dotnet5.4" ]
-    }
+    "netstandard1.6": { }

```

Comment:
Does that imply that `dotnet5.4` is a subset of `netstandard1.6`? If so, then the import element seems redundant. If not, I can't imagine why anyone would import the dependency since it would be too dangerous.